_studio/hevce_hw/Android.mk

```diff
@@ -1,87 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
```

Comment:
How changes to h.265 relates to videowall tm?

Reply:
Sorry, nothing, need to delete this file from this commit